### ISSUE TITLE
Docs: Add copy button code blocks

### ISSUE THREAD

**Issue by Jay-Karia** [eslint/eslint#18769]
### Docs page(s)

One of the page
https://eslint.org/docs/latest/use/getting-started#manual-set-up

### What documentation issue do you want to solve?

A copy button is missing in the code blocks.

### What do you think is the correct solution?

Adding a copy button to the code blocks.
Modifying the `md`: by adding an attribute to code block: `"```shell  copy..."`

### Participation

- [X] I am willing to submit a pull request for this change.

**Comment by Tanujkanti4441** [comment]
Hi @Jay-Karia, thanks for the issue, sounds good to me, can be a helpful feature specially for `command-line-interface` page.

@eslint/eslint-team, should we consider this?

**Comment by amareshsm** [comment]
I am up for this change 👍 .

**Comment by nzakas** [comment]
Sounds good to me. We'll just have to make sure it doesn't show up in the same code blocks that have "Open in Playground" to avoid being too busy.

**Comment by Jay-Karia** [comment]
Here's my take on this issue:
Adds a new component: `copy-button` which would run a script inside it and copies the text passed as parameter.

@amareshsm @nzakas 
Please let me know your opinions.

**Comment by nzakas** [comment]
We have a lot of code examples on the site, and I don't think we want to manually go through and update each one to use a new component. I think it would be better to try to find a way to insert the button just like we do with the "Open in Playground" button now.

**Comment by Jay-Karia** [comment]
Will look at it
